meinberlin/react/livequestions/QuestionUser.jsx

```diff
@@ -1,97 +1,51 @@
-import React from 'react'
-import django from 'django'
+import React, { useEffect, useState, useCallback } from 'react'
+import LikeCard from './LikeCard'
 
-export default class QuestionUser extends React.Component {
-  constructor (props) {
-    super(props)
+const QuestionUser = ({ is_on_shortlist: isOnShortlist, likes, handleLike, id, children, category, hasLikingPermission }) => {
+  const [, setIsOnShortlist] = useState(isOnShortlist)
+  const [likeCount, setLikeCount] = useState(likes.count)
+  const [sessionLike, setSessionLike] = useState(likes.session_like)
 
-    this.state = {
-      is_on_shortlist: this.props.is_on_shortlist,
-      is_live: this.props.is_live,
-      likes: this.props.likes.count,
-      session_like: this.props.likes.session_like
-    }
-  }
+  useEffect(() => {
+    setIsOnShortlist(isOnShortlist)
+  }, [isOnShortlist])
```

Comment:
What is this for?

Reply:
It's for users that are on "shortlists": I've reverted the HTML for this, in case there will be a future ticket to style: https://github.com/liqd/a4-meinberlin/pull/5744/files#diff-049654a90fe27d059ce8bf97f8bfba60e7543bc13b2e847888e1010604fb2462R43-R47